pkg/operators/ebpf/ebpf.go

```diff
@@ -530,9 +530,20 @@ func (i *ebpfInstance) Start(gadgetCtx operators.GadgetContext) error {
 	}
 
 	i.logger.Debugf("creating ebpf collection")
-	collection, err := ebpf.NewCollectionWithOptions(i.collectionSpec, ebpf.CollectionOptions{
+	opts := ebpf.CollectionOptions{
 		MapReplacements: mapReplacements,
-	})
+	}
+
+	// check if the btfgen operator has stored the kernel types in the context
+	if btfSpecI, ok := gadgetCtx.GetVar("kernelTypes"); ok {
```

Comment:
I added the const, actually two of them with the same value. I'm not sure how these two operators could share it, specially because we don't want them to import each other.